py/selenium/webdriver/firefox/webdriver.py

```diff
@@ -114,3 +116,25 @@ def firefox_profile(self):
 
     def set_context(self, context):
         self.execute("SET_CONTEXT", {"context": context})
+
+    @contextmanager
+    def using_context(self, context):
+        """Sets the context that Selenium commands are running in using
+        a `with` statement. The state of the context on the server is
+        saved before entering the block, and restored upon exiting it.
+
+        :param context: Context, may be one of the class properties
+            `CONTEXT_CHROME` or `CONTEXT_CONTENT`.
```

Comment:
Class properties have been added. That change got lost when I had to redo the PR.